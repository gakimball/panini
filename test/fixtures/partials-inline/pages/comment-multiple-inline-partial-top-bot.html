{{!--
{{#*inline "layout-inline-partial-bot"}}
<p>layout-inline-partial-bot content in comment</p>
{{/inline}}
{{#*inline "layout-inline-partial-top"}}
<p>layout-inline-partial-top content in comment</p>
{{/inline}}
--}}
{{#*inline "layout-inline-partial-bot"}}
<p>layout-inline-partial-bot content</p>
{{/inline}}
{{#*inline "layout-inline-partial-top"}}
<p>layout-inline-partial-top content</p>
{{/inline}}
<p>Page Body</p>
{{!--
{{#*inline "layout-inline-partial-bot"}}
<p>layout-inline-partial-bot content in comment bottom page</p>
{{/inline}}
{{#*inline "layout-inline-partial-top"}}
<p>layout-inline-partial-top content in comment bottom page</p>
{{/inline}}
--}}