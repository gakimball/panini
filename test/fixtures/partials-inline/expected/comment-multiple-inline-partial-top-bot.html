<html>
  <body>

<p>layout-inline-partial-top content</p>

    <p>Page Body</p>

<p>layout-inline-partial-bot content</p>

  </body>
</html>
